x-pack/plugins/cases/public/components/links/index.tsx

```diff
@@ -18,7 +18,7 @@ import * as i18n from './translations';
 
 export interface CasesNavigation<T = React.MouseEvent | MouseEvent, K = null> {
   href: K extends 'configurable' ? (arg: T) => string : string;
-  onClick?: (arg: T) => void;
+  onClick: (arg: T) => void;
```

Comment:
made this required, href triggers app reload. we need href for right click > open in new tab